AlphaWallet/Core/Coordinators/WalletBalance/WalletBalanceCoordinator.swift

```diff
@@ -186,7 +186,7 @@ class WalletBalanceCoordinator: NSObject, WalletBalanceCoordinatorType {
     }
 
     private func notifyWalletSummary() {
-        walletSummary.done(on: queue, { [weak self] summary in
+        walletSummary.done(on: .main, { [weak self] summary in
```

Comment:
Doesn't need `.main` since that's the default for PromiseKit, right? But we should reduce our usage of custom queues anyway. Would you modify and then merge?